core/test/utils/fixtures/export/export-000.json

```diff
@@ -10,8 +10,8 @@
                 "uuid": "8492fbba-1102-4b53-8e3e-abe207952f0c",
                 "title": "Welcome to Ghost",
                 "slug": "welcome-to-ghost",
-                "markdown": "You're live! Nice.",
-                "html": "<p>You're live! Nice.</p>",
+                "mobiledoc": "{\"version\":\"0.3.1\",\"markups\":[],\"atoms\":[],\"cards\":[[\"card-markdown\",{\"cardName\":\"card-markdown\",\"markdown\":\"You're live! Nice.\"}]],\"sections\":[[10,0]]}",
```

Comment:
Should these export fixtures be using the 1.0 format or should they be kept in the LTS format? I noticed there were other LTS specific fixtures so I wasn't sure.